studio/src/app/pages/user-profile/user-profile.component.ts

```diff
@@ -1,38 +1,86 @@
 import { Component, OnInit } from '@angular/core';
-import { FormBuilder, FormGroup, ReactiveFormsModule ,Validators } from '@angular/forms';
+import {
+  FormBuilder,
+  FormGroup,
+  ReactiveFormsModule,
+  Validators,
+} from '@angular/forms';
 import { UserProfileService } from './service/user-profile.service';
+import { UserProfile } from '../../core/models/user-profile';
+import { UserToken } from '../../core/models/user-token';
+import { NgClass, NgStyle } from '@angular/common';
 
 @Component({
   selector: 'app-user-profile',
   standalone: true,
-  imports: [ReactiveFormsModule],
+  imports: [ReactiveFormsModule, NgStyle, NgClass],
   templateUrl: './user-profile.component.html',
-  styleUrl: './user-profile.component.scss'
+  styleUrl: './user-profile.component.scss',
 })
 export class UserProfileComponent implements OnInit {
   protected userProfileForm!: FormGroup;
-  private userId : string = '75b2120c-7518-47e7-972b-8b266e51ee59';
+  private userId: string = '75b2120c-7518-47e7-972b-8b266e51ee59';
+  protected isReadonly: boolean = true;
+  private userToken!: UserToken;
+  protected userPhoto!: string;
+
+  constructor(
+    private fb: FormBuilder,
+    private userProfileService: UserProfileService
+  ) {}
 
-  constructor(private fb: FormBuilder, private userProfileService: UserProfileService) {}
-  
   ngOnInit(): void {
-    this.getUser();
     this.createForm();
+    this.getUser();
   }
 
-  createForm(){
+  createForm() {
```

Comment:
Move this, create form inside  Get profile subscribe. (this.userProfileService.GetProfile(this.userId).subscribe).
Instead of create with empty string set the value using x.userName for example and remove setValue methods